src/window/window_wrapper.rs

```diff
@@ -175,6 +178,25 @@ impl WinitWindowWrapper {
                 self.mouse_manager.enabled = mouse_enabled
             }
             WindowCommand::ListAvailableFonts => self.send_font_names(),
+            WindowCommand::Bell => {
+                unsafe {
+                    #[cfg(target_os = "macos")]
+                    NSBeep();
+                    #[cfg(target_os = "windows")]
+                    MessageBeep(MB_OK);
+                }
+                #[cfg(not(any(target_os = "macos", target_os = "windows")))]
```

Comment:
I think this can be enabled on all platforms. And used instead of the platform bells if defined.